src/main/java/practice/StreamPractice.java

```diff
@@ -23,7 +33,12 @@ public int findMinEvenNumber(List<String> numbers) {
      * But before that subtract 1 from each element on an odd position (having the odd index).
      */
     public Double getOddNumsAverage(List<Integer> numbers) {
-        return 0D;
+        return (double)Math.round(numbers.stream()
```

Comment:
```suggestion
        return (double)Math.round(numbers.stream()
```
why do you use round() method here?